endpointGroup.json

```diff
@@ -75,6 +75,9 @@
 				"policies": {
 					"ref": "policy"
 				},
+				"rules": {
+					"ref": "rule"
```

Comment:
Just to keep links/linksets symmetric, can we add a link from rule to EPG along with link-set from EPG to rule?